src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+spider.style.position = 'static';
+wall.style.display = 'flex';
+wall.style.alignItems = 'center';
+wall.style.justifyContent = 'center';
```

Comment:
:smile: 
Yeah, it could be easily solved with pure CSS. But the idea is to solve it with JS - use wall and spider size